internal/namespace/manager.go

```diff
@@ -29,7 +29,7 @@ type Manager interface {
 	// ReadNamespaceAndTypes reads a namespace definition, version, and type system and returns it if found.
 	ReadNamespaceAndTypes(ctx context.Context, nsName string, revision decimal.Decimal) (*v0.NamespaceDefinition, *NamespaceTypeSystem, error)
 
-	// Closes the namespace manager, disposing of any resources.
+	// Close closes the namespace manager, disposing of any resources.
```

Comment:
Are we going to enforce linting rules on the e2e code? Because it's a separate project almost all the tooling doesn't run on this code.

Reply:
this is actually in the main module, not e2e

but yeah we should probably run the linters over this too

Reply:
>this is actually in the main module, not e2e

Yep, the fact that this is a lint fix just reminded me that the other files are not being checked.